titus-common-ext/kube/src/test/java/com/netflix/titus/ext/kube/clustermembership/connector/KubeExternalResource.java

```diff
@@ -40,7 +39,7 @@ protected void before() throws Throwable {
                 .standard()
                 .setBasePath(String.format("http://%s:7001", kubeServer))
                 .build();
-        client.getHttpClient().setReadTimeout(0, TimeUnit.SECONDS); // infinite timeout
+        client.setReadTimeout(0); // infinite timeout
```

Comment:
same as above

Reply:
I do not know why it was set like that. I am just keeping the original behavior.
Even if not needed, I do not want to remove it now.